ledger/sandbox/src/main/scala/com/digitalasset/platform/sandboxnext/Runner.scala

```diff
@@ -81,11 +81,13 @@ class Runner {
     }
 
     val timeProviderType = config.timeProviderType.getOrElse(TimeProviderType.Static)
-    val timeServiceBackend = timeProviderType match {
+    val (timeServiceBackend, heartbeatMechanism) = timeProviderType match {
       case TimeProviderType.Static =>
-        Some(TimeServiceBackend.simple(Instant.EPOCH))
+        val backend = TimeServiceBackend.observing(TimeServiceBackend.simple(Instant.EPOCH))
+        (Some(backend), backend.changes)
       case TimeProviderType.WallClock =>
-        None
+        val clock = Clock.systemUTC()
+        (None, new RegularHeartbeat(clock, 1.second))
```

Comment:
Shouldn't this be configurable or at least specified by a constant?

Reply:
Good call. I'll pull out a constant.